src/lib/widgets/bottom-bar/ui/mini-player.svelte

```diff
@@ -25,33 +21,5 @@
   }}
 >
   <img src={podcast?.coverUrl} width="40" alt="" />
-  <div class="flex justify-between">
-    <IconButton
-      name="Replay the last 10 seconds"
-      icon={IconReplay10}
-      class="w-9 h-9"
-      iconClass="w-7 h-7"
-      on:click={(e) => {
-        e.stopPropagation();
-        move(-10);
-      }}
-    />
-    <PlaybackButton
-      playing={!$paused}
-      on:click={(e) => {
-        e.stopPropagation();
-        toggleGlobalPlayback();
-      }}
-    />
-    <IconButton
-      name="Skip forward 30 seconds"
-      icon={IconForward30}
-      class="w-9 h-9"
-      iconClass="w-7 h-7"
-      on:click={(e) => {
-        e.stopPropagation();
-        move(30);
-      }}
-    />
-  </div>
+  <PlaybackControlButtons />
```

Comment:
thoughts: I wonder if this dependency is justified. I think that this duplication here does not cause too much trouble, but we retain the ability of customizing the appearance of the Now Playing page without breaking the mini player. Namely, the `prominent` prop makes sense for the playback button, but makes less sense for the full controls.

suggestion: keep this code as it was